reader/panels/container_panel_view.js

```diff
@@ -33,12 +33,12 @@ ContainerPanelView.Prototype = function() {
 
   this.render = function() {
     // Hide the whole tab if there is no content
-    if (this.getContainer().getLength() === 0) {
-      this.hideToggle();
-      this.hide();
-    } else {
+    if (this.getContainer().hasContent(this.config.type)) {
```

Comment:
@gnott I've added "this.config.type" to the PR, I missed it when did the fork, sorry.

Reply:
That fills in the missing part, thanks for this @DavidMorenoCortina!